src/averageAges.js

```diff
@@ -38,6 +48,18 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const ifWithChildren = people.filter((person) => withChildren
+    ? people.find(child => child.mother === person.name) && person.sex === 'f'
```

Comment:
```suggestion
    ? people.find(child => child.mother === person.name)
```
Те що вона мама вже означає що це жінка